code/operators/solution/operators.sol.cpp

```diff
@@ -72,11 +55,11 @@ class Fraction {
   int m_num, m_denom;
 };
 
-void printAndCheck(std::string const & what, Fraction const & result, Fraction const & expected) {
+void printAndCheck(const std::string & what, const Fraction & result, const Fraction & expected) {
```

Comment:
I would really keep using const of the right of the type, as it's how it is suppose to work. For me const on the left should even be deprecated and ultimately forbidden

Reply:
Ah yes, this came back in because `operators.cpp` had west-const. I fill fix it.

If you want to have consistent formatting of the `const` keyword, maybe propose a `.clang-format` file. It can rearrange the const placement. I don't care so much, because I am really fine with west-const. It shows me `const` first, before the data type (or `auto`), which is more important to me.